rules/rules-reviewed/azure/azure-file-system.windup.xml

```diff
@@ -57,7 +58,7 @@
                         Any usage of the file system on the application will require reconfiguration or, in rare cases, architectural changes.
                         ]]>
                     </message>
-                    <link title="Relative path found in the application" href="https://docs.microsoft.com/azure/developer/java/migration/migrate-spring-boot-to-app-service#determine-whether-and-how-the-file-system-is-used"/>
+                    <link title="Relative path found in the application" href="https://learn.microsoft.com/en-us/azure/developer/java/migration/migrate-spring-cloud-to-azure-spring-apps?pivots=sc-standard-tier#determine-whether-and-how-the-file-system-is-used"/>
```

Comment:
@showpune Hi!
Sorry I can still see some `en-us` references in some URLs: do you mind changing all of the URLs to fulfill Antonio's request about internationalization?
Thanks.